acceptance/clients/conditions.go

```diff
@@ -147,7 +147,7 @@ func IsReleasesAbove(t *testing.T, release string) bool {
 func IsReleasesBelow(t *testing.T, release string) bool {
 	current_branch := getReleaseFromEnv(t)
 
-	if current_branch != "master" && current_branch < release {
+	if current_branch != "master" || current_branch < release {
```

Comment:
This fix allows to run tests on master branch only while using `clients.SkipReleasesBelow(t, "master")`